airbyte-workers/src/main/java/io/airbyte/workers/DefaultSyncWorker.java

```diff
@@ -82,6 +88,21 @@ public DefaultSyncWorker(final Source<T> source,
 
       destination.notifyEndOfStream();
 
+      final boolean shouldNormalize = Optional
```

Comment:
you could move that code outside and pass in an actual normalizer or a noop. WDYT?

Reply:
i'll try it and see what it looks lik.e